### PR TITLE
Add ComicBookInfo metadata provider

### LINKED ISSUE #35: Support for ComicBookInfo metadata (ComicBookLover)
Add support for ComicBookLover's ComicBookInfo format (not to be confused with ComicRack's ComicInfo format).

### PR DESCRIPTION
**Pull request by MinecraftPlaye**
The ComicBookInfo format stores metadata within the zip archives
comment. As System.IO.Compression does not expose the zip archives
comment, SharpCompress is used instead.

Reads in all fields the ComicBookInfo defines and saves as many as
possible in Jellyfin. Additional fields can be saved once support
for these is added in the Jellyfin Server project.

Closes  #35 
Depends on #34 as it should use the interfaces provided by that PR.

### CODE REVIEW

File: Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                using Stream stream = File.OpenRead(path);
+                // not yet async: https://github.com/adamhathcock/sharpcompress/pull/565
+                using var archive = ZipArchive.Open(stream);
+
+                if (archive.IsComplete)
+                {
+                    var volume = archive.Volumes.First();
+                    if (volume.Comment != null)
+                    {
+                        var jsonStream = new MemoryStream(Encoding.UTF8.GetBytes(volume.Comment));
```
Comment (crobibero) [inline Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs:52]:
```suggestion
                        using var jsonStream = new MemoryStream(Encoding.UTF8.GetBytes(volume.Comment));
```

File: Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                        var comicBookMetadata = await JsonSerializer.DeserializeAsync<ComicBookInfoFormat>(jsonStream, new JsonSerializerOptions()
```
Comment (crobibero) [inline Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs:53]:
```suggestion
                        var comicBookMetadata = await JsonSerializer.DeserializeAsync<ComicBookInfoFormat>(jsonStream, new JsonSerializerOptions()
```

Use `JsonDefaults.Options`. NOTE: this must not be modified.

Comment (MinecraftPlaye) [reply Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs:53]: So there is nothing I should change there?

Comment (crobibero) [reply Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs:53]: The property you get from `JsonDefaults.Options`must not be modified, but you should use it. It already has string number parsing enabled, so no changes needed!

Comment (MinecraftPlaye) [reply Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs]: Ah, ok. Done :)

**Review by crobibero — APPROVED** [review]

**Pull request merged**